### DISCUSSION TITLE
GitHub achievements suddenly not visible

### DISCUSSION THREAD

**Discussion by Curzsu** [community/community#57053] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I own an achievement which names 'Starstruck'.

My Github achievements were still visible at beginning , but then I accidentally hide the achievements, and now no matter how I try to set it, I can't restore it.

**Before:**
![image](https://github.com/community/community/assets/86864691/60667087-5988-4d4e-9314-59d32854de90)

**Now**:
![image](https://github.com/community/community/assets/86864691/a654cd6d-3fa2-4c59-bf3f-a3f61648b637)

**My setting:**
<img width="238" alt="0e3664d275371a7e103878bdd92d3af" src="https://github.com/community/community/assets/86864691/4843b72c-39ca-4637-858a-3d4dfe6a9b75">

I want to change the visibility to visible again. So what measures do I need to take to solve this problem?

**Comment by radar-blip** [comment, marked as answer]
Try this link to unhide them: https://github.com/Curzsu?tab=achievements
You should see something like this:

![gitty](https://github.com/community/community/assets/39196767/7e844bcd-29a0-48b7-adc4-ec9b3da8f745)

**Reply by Curzsu** [reply]
Thank you! the problem is solved.

**Reply by radar-blip** [reply]
Anytime. Happy to help!!
